postgresql/config/functions.sh

```diff
@@ -59,13 +63,14 @@ master_ready() {
 bootstrap_replica_via_pg_basebackup() {
   echo 'Bootstrapping replica via pg_basebackup from leader '
 
-  rm -rf {{pkg.svc_data_path}}/*
-  pg_basebackup --pgdata={{pkg.svc_data_path}} --xlog-method=stream --dbname='postgres://{{cfg.replication.name}}@{{svc.leader.sys.ip}}:{{cfg.port}}/postgres'
+  # TODO: guard this rm -rf
```

Comment:
What kind of guard are you imagining?  If there isn't a blocker, let's add it now.  This `rm -rf` rightfully makes us nervous but it is a necessary evil with pg_basebackup.

Reply:
tl;dr - removed that comment